code/datums/id_trim/jobs.dm

```diff
@@ -473,10 +492,15 @@
 		ACCESS_HOP,
 		)
 	job = /datum/job/cook
+	honorific = "Head Chef"
```

Comment:
Why is "Cook" a Head Chef and actual "Chef" just Chef?

Reply:
Chef is already the head of cooks too

Reply:
![image](https://github.com/user-attachments/assets/964b6078-c495-4315-b291-131fa862d0b7)

Nuts.